minecode/visitors/github.py

```diff
@@ -179,3 +181,26 @@ def json_serial_date_obj(obj):
     """JSON serializer for date object"""
     if obj and isinstance(obj, (datetime, date)):
         return obj.isoformat()
+
+
+# Indexing GitHub PURLs requires a GitHub API token.
+# Please add your GitHub API key to the `.env` file, for example: `GH_TOKEN=your-github-api`.
+@priority_router.route('pkg:github/.*')
+def process_request_dir_listed(purl_str):
```

Comment:
Is there a test that tests github purls or is that handled by the `test_map_fetchcode_supported_package` test?

Reply:
All the PURLs supported by FetchCode use this single function, `map_fetchcode_supported_package`, since FetchCode exposes the single interface, i.e., `package.info` to retrieve the package metadata for all PURL. 

We have extensive tests for these GitHub PURLs in FetchCode, testing all the versions of this package ever released. Please see https://github.com/nexB/fetchcode/blob/ff0bad32881464afbfa6f70a65909ed0170e94e5/tests/test_package.py#L100-L501